client/src/procedures/types.rs

```diff
@@ -134,6 +135,13 @@ impl From<String> for ProcedureOutput {
     }
 }
 
+impl From<bip39::Mnemonic> for ProcedureOutput {
+    fn from(m: bip39::Mnemonic) -> Self {
+        // mnemonic secret is leaked; ProcedureOutput should be zeroized
```

Comment:
Should we make it Zeroize/ZeroizeOnDrop then?

Reply:
Remove the comment then?

Reply:
Fixed comment